src/containers/artist-dashboard-page/ArtistDashboardPage.tsx

```diff
@@ -246,6 +246,12 @@ export class ArtistDashboardPage extends Component<
     TotalPlaysChart.preload()
   }
 
+  componentDidUpdate() {
+    if (!this.props.account?.is_creator) {
+      this.props.goToRoute(TRENDING_PAGE)
```

Comment:
is there a possibility this runs a ton of times before unmount?

Reply:
Theoretically I think it could, but I didn't see anything weird in my testing